src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -163,8 +165,18 @@ onMounted(async () => {
 	);
 
 	updateHeaderPosition();
-	window.addEventListener('scroll', updateHeaderPosition);
+
+	window.addEventListener('scroll', _throttle(() => {
```

Comment:
The throttle call returns a new instance, so it won't be removed like expected. You need to use a variable, like here: https://github.com/kiva/ui/blob/c85874e0adc6abc2e8bd4d271185104862703175/src/composables/useIsMobile.js#L18